src/libgeodecomp/geometry/coord.h

```diff
@@ -624,6 +642,19 @@ class Coord<3>
         c[2] = int(other[2]);
     }
 
+    inline explicit Coord(std::initializer_list<int> list)
+    {
+        int i = 0;
+        for (int coord : list){
+            c[i] = coord;
+            if (i > 2)
```

Comment:
Please add curly braces around the statement after the conditional. LGD is using 1TBS as coding style.